contracts/examples/crowdfunding-esdt/src/lib.rs

```diff
@@ -82,7 +78,8 @@ pub trait Crowdfunding {
 				let esdt_balance = self.get_esdt_balance();
 
 				self.set_esdt_balance(&BigUint::zero());
-				self.pay_esdt(&esdt_token_name, &esdt_balance, &caller);
+				self.send()
+					.direct(&caller, &esdt_token_name, &esdt_balance, &[]);
```

Comment:
I think writing `b""` instead of `&[]` would be more suggestive that we're sending a transaction with an empty message.